src/transformers/models/mpnet/modeling_tf_mpnet.py

```diff
@@ -552,12 +579,10 @@ class PreTrainedModel
         """
         raise NotImplementedError
 
-    # Copied from transformers.models.bert.modeling_tf_bert.TFBertMainLayer.call
     def call(
         self,
         input_ids=None,
         attention_mask=None,
-        token_type_ids=None,
```

Comment:
I understand MPNet's implementation has some `token_type_ids` it doesn't use, but I'd leave them for now here until there is a general fix (that also deals with the PyTorch implementation). The tokenizer still return those token_type_ids so this would cause problem if a user feeds the output of a tokenizer to one of those models.

Reply:
`token_type_ids` is not in the PyTorch implementation, so I think the Tokenizer should be fixed in same time than the TF model.